framework/build.gradle

```
@@ -152,3 +152,7 @@ bintray {
         }
     }
 }
+
+dependencies {
+    implementation 'androidx.webkit:webkit:1.3.0'
+}
```

```suggestion
}

```

I think we always add a new line to the end of our files. But not sure about the gradle extension files.